WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -893,11 +893,14 @@ local methods = {
       notCheckable = true,
       func = function() OptionsPrivate.ExportToString(self.data.id) end
     });
-    tinsert(self.menu, {
-      text = L["Export debug table..."],
-      notCheckable = true,
-      func = function() OptionsPrivate.ExportToTable(self.data.id) end
-    });
+
+    if WeakAuras.buildType == "dev" then
```

Comment:
test123

Reply:
So i was thinking for some people like wa discord moderators it could be usefull to still have access to this while not using dev version
For that use case it would be nice to have a command toggling that like `/wa debug`